content/guides/models/track/runs/multiple-runs-per-process

```diff
@@ -0,0 +1,147 @@
+# Multiple Active Runs in One Process (Reinit Behavior)
+
+**Quick Reference**  
+- **`return_previous`**: Return the existing run; no new run is created.  
+- **`finish_previous`**: Finish (end) the old run, then create a new one.  
+- **`create_new`**: Create a new run without finishing the old one—great for parallel or interleaved runs, but watch out for integration compatibility.
+
+---
+
+By default, W&B assumes each Python process has only **one active run** at a time. Calling `wandb.init()` again, in the default scenario, will either return the same run or finish the old run before starting a new one—depending on the configuration. However, there are workflows where you might want **multiple runs** in the same Python process:
+
+- Keeping a single “primary” run active throughout a script while spinning up short-lived “secondary” runs for evaluations or sub-tasks.  
+- Orchestrating sub-experiments in a single file.  
+- Logging from one “master” process to several runs that represent different tasks or time periods.
+
+W&B provides a **`reinit`** setting to control how `wandb.init()` behaves when there’s an existing active run. Below, we detail each `reinit` option and show how to maintain multiple concurrent runs—finishing them at different times without interference.
+
+---
+
+## Overview of `reinit` Options
+
+Use the `reinit` argument when calling `wandb.init()`, or set it via environment variable (e.g., `WANDB__INIT__REINIT`). The valid options:
```

Comment:
The environment variable is wrong. It is just `WANDB_REINIT`.

I think it's important to call out that `reinit` is a setting, and any way of configuring settings works.

> Set the `reinit` setting in any of the following ways:
>
> * Passing it as the `reinit` argument to `wandb.init()` or via the `settings` argument
> * Passing it via `settings` to `wandb.setup()` (applies to all `wandb.init()` calls)
> * Setting the `WANDB_REINIT` environment variable (applies to all `wandb.init()` calls)

There is another way, which is to set it in the config file, but I couldn't find any documentation about that to link out to.